net.lutris.Lutris.yaml

```diff
@@ -1,11 +1,10 @@
 ---
-app-id: org.lutris.Lutris
+app-id: net.lutris.Lutris
 runtime: org.gnome.Platform
 runtime-version: '3.30'
 sdk: org.gnome.Sdk
 command: lutris
-rename-desktop-file: lutris.desktop
-rename-appdata-file: lutris.appdata.xml
+rename-appdata-file: net.lutris.Lutris.appdata.xml
```

Comment:
The `rename-appdata-file` line should not be needed anymore as well. :-)

Reply:
Anyway, there is currently a small issue with Lutris AppData file having bad desktop file ID. See https://github.com/lutris/lutris/pull/1808.

Reply:
Is the `rename-appdata-file` line somehow useful anymore?

Reply:
Unsure! Haven't tried to build it since I pushed that forwards. We may not need it.

Biggest blocker is GnomeDesktop namespace not being there when running.

Reply:
GnomeDesktop is included in runtime https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/master/elements/core/gnome-desktop.bst

Reply:
Its not, what's in the runtime is defined in this [file][1]. `gnome-desktop` is a highly unstable component (on purpose) that [won't be part][2] of the runtime. Please bundle it, all the other apps do so as well.

[1]: https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/9fa4f664f0c562bd11432be8eee8733773f10c6c/elements/sdk-platform.bst
[2]: https://gitlab.gnome.org/GNOME/gnome-build-meta/issues/125#note_447760